packages/cli/subtasks/pick-contract.js

```diff
@@ -7,7 +7,7 @@ subtask(SUBTASK_PICK_CONTRACT, 'Pick contract to interact with').setAction(
   async (taskArguments, hre) => {
     const contracts = Object.keys(hre.deployer.deployment.general.contracts);
 
-    _prioritizeTarget(contracts, 'Synthetix');
+    _prioritizeTarget(contracts, 'SNXTokenModule');
```

Comment:
Should probably remove this, since we want this CLI to be non-Synthetix specific.

Reply:
Removed